cmd/balancesheet/download.go

```
@@ -0,0 +1,214 @@
+		return
+	}
+
+	curYear := time.Now().Year()
+	if endYearStr != "" {
+		endYear, err = strconv.Atoi(endYearStr)
+		if err != nil {
+			slog.Error("genStartEndDate", "end-year", endYearStr, "error", err)
+			return
+		}
+
+		startDate = fmt.Sprintf("%d-01-01", endYear)
```

golangci-lint (2.12.2)

[error] ineffectual assignment to startDate

(ineffassign)